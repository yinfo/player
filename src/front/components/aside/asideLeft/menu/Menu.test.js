// @flow

import React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router';
import Menu from './Menu';
import { navigation } from '../../../../models/navigation';

describe('Menu component', () => {
  const views = navigation.sideMenu[0];

  it('renders as expected', () => {
    const component = shallow(
      <MemoryRouter>
        <Menu
          headerTitle="header title"
          headerBackColor="#F1F2F3"
          activeView={views.menus[0].name}
          views={views.menus}
          initialCollapseState={false}
        />
      </MemoryRouter>,
    );
    expect(component).toMatchSnapshot();
  });
});
