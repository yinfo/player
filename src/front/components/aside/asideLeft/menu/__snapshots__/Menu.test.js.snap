// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component renders as expected 1`] = `ShallowWrapper {}`;
